.azuredevops/deploy-platform-bootstrap.yaml

```diff
@@ -112,7 +112,16 @@ extends:
                 path: infra/common/network
                 serviceConnectionVariableName: "subscriptionName"
                 resourceGroupName: $(virtualNetworkResourceGroup)
+                postDeployScriptsList:
+                  - displayName: Link the VNet to Hub
```

Comment:
_Doesn't the VNET need to exist first mate ;)_

Reply:
Yes, that was just to test the script.. corrected now. :)